upgrade/rendering-changes/list.md

```diff
@@ -0,0 +1,135 @@
+---
+title: List of Versions
+description: See which versions have rendering changes so you can take them into account when upgrading.
+page_title: List of Versions with Rendering Changes
+slug: versions-with-rendering-changes
+position: 0
+---
+
+# List of Telerik UI for Blazor Versions with Rendering Changes
+
+
+As the Telerik UI for Blazor matures we will continuously optimize the rendering and the usage of CSS classes. The rendering changes will affect the styling of the components if the business application overrode the built-in CSS rules. This article will track the version where such changes are introduced.
+
+* [3.4.0]({%slug changes-in-3-4-0%})
+
+## Good Styling Practices
+
+@[template](/_contentTemplates/common/good-styling-practices.md#good-styling-practices)
+
+>caption Provide some custom CSS rules to the header and cells of the Grid. In the `Advisable` tab you can see the good practices in action. 
+
+<div class="skip-repl"></div>
+````Advisable
```

Comment:
How about "recommended" and "not recommended" ? "Inadvisable" sounds a bit exotic.

If you need to use single words, then perhaps "prefer" + "avoid"